suite-native/graph/src/components/TransactionEvent.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { runOnJS, useDerivedValue, withSpring } from 'react-native-reanimated';
+
+import { Circle, Group } from '@shopify/react-native-skia';
+
+import { EventComponentProps } from '@suite-native/react-native-graph/src/LineGraphProps';
+import { Color } from '@trezor/theme';
+import { useNativeStyles } from '@trezor/styles';
+import { GroupedBalanceMovement } from '@suite-common/graph';
+
+import { GraphContextProvider } from './GraphContextProvider';
+import { TransactionEventTooltipProps } from './TransactionEventTooltip';
+
+type EventVariant = 'positive' | 'negative';
+
+type TransactionEventProps = EventComponentProps &
```

Comment:
You could just import `GraphTransactionEventPayload` from `Graph.tsx` and extend it with `EventComponentProps` to avoid duplication.

Reply:
Sure! For next time, it's better to move the type somewhere where it doesn't create circular deps instead of duplicating it. So for this case, you would move it here and import in `Graph.tsx` or move to some type file which could export to all places where it's needed